src/Teapot.Web/Startup.cs

```diff
@@ -0,0 +1,59 @@
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+namespace Teapot.Web
+{
+    public class Startup
+    {
+        public Startup(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+
+        public IConfiguration Configuration { get; }
+
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddCors(options =>
+            {
+                options.AddPolicy(Constants.CorsPolicy, builder => builder
+                    .AllowAnyHeader()
+                    .AllowAnyOrigin()
+                    .AllowAnyMethod()
+                    .WithExposedHeaders(new[]
+                    {
+                        "Link", // 103
+                        "Content-Range", // 206
+                        "Location", // 301, 302, 303, 305, 307, 308
+                        "WWW-Authenticate", // 401
+                        "Proxy-Authenticate", // 407
+                        "Retry-After" // 429
+                    }));
+            });
+
+            services.AddControllersWithViews();
+        }
+
+        public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
+        {
+            if (env.IsDevelopment())
+            {
+                app.UseDeveloperExceptionPage();
+            }
+
+            app.UseHsts();
+            app.UseHttpsRedirection();
```

Comment:
Do we want to do that? This will mean it's not possible to test on HTTP-only, but I'm not sure if that's desired.

Reply:
ASP.NET Core provides local development SSL certificates on all supported platforms, so it should be fine. When deployed to App Service, I think it'd be wise to have HTTPS only enabled at the IIS level.

Reply:
which makes sense.